### PR TITLE
Small code improvement.

### PR DESCRIPTION
**Pull request by ptondereau**
Just some basic proofreading of code done quick

### CODE REVIEW

File: src/api/rule.rs [hunk 1]
```diff
@@ -77,13 +86,9 @@ impl Rule {
...
-        for marker in &self.markers {
-            if marker.name.as_str() == name {
-                return Some(marker);
-            }
-        }
-
-        None
+        self.markers
+            .iter()
+            .find(|&marker| marker.name.as_str() == name)
```
Comment (ptondereau) [inline src/api/rule.rs:91]: I think it makes it easier to read and it doesn't affect perf

File: src/build.rs [hunk 2]
```diff
@@ -337,7 +346,7 @@ fn build_test_file(file: DirEntry) -> std::io::Result<(String, RuleSet)> {
         .unwrap()
         .to_string()
         .replace(".yml", "")
-        .replace("-", "_");
+        .replace('-', "_");
```
Comment (ptondereau) [inline src/build.rs:349]: Performing these methods using a `char` is faster than using a `str`.

File: src/html/mod.rs [hunk 3]
```diff
@@ -1,9 +1,12 @@
...
+use crate::html::TokenType::{
+    CommentToken, DoctypeToken, EndTagToken, ErrorToken, SelfClosingTagToken, StartTagToken,
+    TextToken,
+};
 use std::io::Read;
 use std::str;
 use std::string::ToString;
 
-#[derive(Debug, Clone, Copy, PartialEq)]
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
```
Comment (ptondereau) [inline src/html/mod.rs:9]: If a type `T` derives `PartialEq` and all of its members implement `Eq`, then `T` can always implement `Eq`. Implementing `Eq` allows `T` to be used in APIs that require `Eq` types. It also allows structs containing `T` to derive `Eq` themselves.

**Comment by joelwurtz** [comment]
fixed in https://github.com/redirectionio/libredirectionio/commit/03a5289534fc61cb73e2b82fe89cbf4a0f635f3d

Sorry for not taking your pr directly but there way too many conflict due to the format,  as we have a different rustfmt from the default, to avoid this problem in the future we added the rustfmt file at the root repository so it should not happen again.

Anyway thanks for you contribution

**Pull request closed without merging**